data/engines.txt

```diff
@@ -223,7 +223,7 @@ outfit "X5200 Ion Steering"
 	"turn" 2174
 	"turning energy" 3.5
 	"turning heat" 7.3
-	description "The X5200 is a steering system so large and powerful that only a handful of ships can install it. This means that this steering system is rarely sold due to its limited market."
+	description "The X5200 is a steering system so large and powerful that only a handful of ships can install it. It is rarely sold due to its limited market."
```

Comment:
`it. It is` is all really similar.

Reply:
"The X5200 is a steering system so large and powerful that only a handful of ships can install it. Because of its size, it is rarely sold due to its limited market."

just adds 4 words in-between, but it does make the flow better

Reply:
How about "Due to its limited market, it is produced in small numbers"? Or delete the second sentence entirely.